EpamWebSiteTestingFramework/Epam.TestAutomation.Pages/PageObjects/Pages/JobListingsPage.cs

```diff
@@ -0,0 +1,114 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Core.Helper;
+using Epam.TestAutomation.Core.Utils;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Pages.PageObjects.Pages;
+
+public class JobListingsPage : BasePage
+{
+    private JobSearchResultPage _resultPage;
```

Comment:
again. references to other pages. In page we can have references only to components